pkg/sql/logictest/testdata/logic_test/fk

```diff
@@ -657,23 +657,9 @@ ALTER TABLE delivery DROP COLUMN "item"
 statement ok
 DROP INDEX products@products_upc_key CASCADE
 
-skipif config local-legacy-schema-changer
-statement error "products" is referenced by foreign key from table "customer reviews"
-DROP TABLE products
-
-# Legacy schema changer orders dependent fks differently, but
-# semantically the same error.
-onlyif config local-legacy-schema-changer
 statement error "products" is referenced by foreign key from table "orders"
 DROP TABLE products
 
-skipif config local-legacy-schema-changer
-statement error referenced by foreign key from table "customer reviews"
-DROP TABLE products RESTRICT
-
-# Legacy schema changer orders dependent fks differently, but
-# semantically the same error.
-onlyif config local-legacy-schema-changer
```

Comment:
Changes in `scbuild` now make these behave in the same way for both schema changer implementations.